### PR TITLE
CreateMultiConfigurationProject_US_041_Test

### PR DESCRIPTION
**Pull request by ksutitova**
https://trello.com/c/qR3P9QDh/511-at041006-createmulticonfigurationprojectus041test-tc041006createmulticonfigurationprojecttest

### CODE REVIEW

File: src/test/java/CreateMultiConfigurationProject_US_041_Test.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        String expectedResult = "Mcproject";
+
+        createMCProject(PROJECT_NAME);
+
+        String actualResult = getDriver().findElement(By.xpath("//div/ul/li/a[@href='/job/Mcproject/']")).getText();
+        Assert.assertEquals(actualResult, expectedResult);
+
+        deleteMCProject();
+    }
+
```
Comment (OVFgit) [inline src/test/java/CreateMultiConfigurationProject_US_041_Test.java:35]: Empty line

File: src/test/java/CreateMultiConfigurationProject_US_041_Test.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        getDriver().findElement(By.xpath("//span[text()='Multi-configuration project']")).click();
+        getDriver().findElement(By.id("ok-button")).click();
+        getDriver().findElement(By.xpath("//button[normalize-space()='Save']")).click();
+    }
+
+    private void deleteMCProject() {
+        getDriver().findElement(By.linkText("Delete Multi-configuration project")).click();
+        getDriver().switchTo().alert().accept();
+    }
+
```
Comment (OVFgit) [inline src/test/java/CreateMultiConfigurationProject_US_041_Test.java:22]: Empty line

**Review by OVFgit — APPROVED** [review]

File: src/test/java/CreateMultiConfigurationProject_US_041_Test.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    @Test
+    public void TC_041_006_CreateMultiConfigurationProjectTest() {
+        String expectedResult = "Mcproject";
+
+        createMCProject(PROJECT_NAME);
+
+        String actualResult = getDriver().findElement(By.xpath("//div/ul/li/a[@href='/job/Mcproject/']")).getText();
```
Comment (azellus-tercia) [inline src/test/java/CreateMultiConfigurationProject_US_041_Test.java:30]: You can use PROJECT_NAME in the line of xpath.

**Review by azellus-tercia — APPROVED** [review]

File: src/test/java/CreateMultiConfigurationProject_US_041_Test.java (hunk 3 above)
Comment (ksutitova) [reply src/test/java/CreateMultiConfigurationProject_US_041_Test.java:30]: Thanks

**Pull request merged**